ocaml-cohttp-eio/lib/scenarios.ml

```diff
@@ -0,0 +1,207 @@
+open Eio.Std
+open Cohttp_eio
+open Utils
+
+let make_request ~port ~path ?query ~sw ~net () =
```

Comment:
It would be neater to move the `Switch.run` inside here, to show that the request is finished by the time the function returns. It would also remove some clutter from the users of the function.

(https://github.com/ocaml-multicore/eio?tab=readme-ov-file#switches-1)

Reply:
Thank you. changed this!